soccer/src/soccer/planning/tests/conversion_tests.cpp

```diff
@@ -13,62 +13,65 @@ namespace planning {
 
 namespace testing {
 
-// NOLINTNEXTLINE
-TEST(RosConversions, EmptyMotionCommand) { test_lossless_convert_cpp_value(EmptyMotionCommand{}); }
-
-// NOLINTNEXTLINE
-TEST(RosConversions, PathTargetMotionCommand) {
-    PathTargetMotionCommand path_target{LinearMotionInstant(Point(1.0, 2.0), Point(3.0, 4.0)),
-                                        FacePoint{Point(1.0, 2.0)}};
-    test_lossless_convert_cpp_value(path_target);
-}
-
-// NOLINTNEXTLINE
-TEST(RosConversions, WorldVelMotionCommand) {
-    test_lossless_convert_cpp_value(WorldVelMotionCommand{Point(1.0, 2.0)});
-}
-
-// NOLINTNEXTLINE
-TEST(RosConversions, PivotMotionCommand) {
-    test_lossless_convert_cpp_value(PivotMotionCommand{Point(1.0, 2.0), Point(3.0, 4.0)});
-}
-
-// NOLINTNEXTLINE
-TEST(RosConversions, SettleMotionCommand) {
-    test_lossless_convert_cpp_value(SettleMotionCommand{Point(1.0, 2.0)});
-}
-
-// NOLINTNEXTLINE
-TEST(RosConversions, CollectMotionCommand) {
-    test_lossless_convert_cpp_value(CollectMotionCommand{});
-}
-
-// NOLINTNEXTLINE
-TEST(RosConversions, GoalieIdleMotionCommand) {
-    test_lossless_convert_cpp_value(GoalieIdleMotionCommand{});
-}
-
-// NOLINTNEXTLINE
-TEST(RosConversions, LineKickMotionCommand) {
-    test_lossless_convert_cpp_value(LineKickMotionCommand{Point(1.0, 2.0)});
-}
-
-// NOLINTNEXTLINE
-TEST(RosConversions, InterceptMotionCommand) {
-    test_lossless_convert_cpp_value(InterceptMotionCommand{Point(1.0, 2.0)});
-}
-
-// NOLINTNEXTLINE
-TEST(RosConversions, MotionCommand) {
-    PathTargetMotionCommand path_target{LinearMotionInstant(Point(1.0, 2.0), Point(3.0, 4.0)),
-                                        FacePoint{Point(1.0, 2.0)}};
-    test_lossless_convert_cpp_value(MotionCommand{EmptyMotionCommand{}});
-    test_lossless_convert_cpp_value(MotionCommand{path_target});
-    test_lossless_convert_cpp_value(MotionCommand{WorldVelMotionCommand{Point(1.0, 2.0)}});
-    test_lossless_convert_cpp_value(
-        MotionCommand{PivotMotionCommand{Point(1.0, 2.0), Point(3.0, 4.0)}});
-    test_lossless_convert_cpp_value(MotionCommand{InterceptMotionCommand{Point(1.0, 2.0)}});
-}
+// TODO(Kevin): add conversion test for new MotionCommand
```

Comment:
I'm guessing that this is a low priority task right now?

Reply:
yes if you could card this that'd be great